src/main/java/org/maxgamer/rs/core/server/Server.java

```diff
@@ -154,7 +154,7 @@
      */
     private Scheduler scheduler;
 
-    public Server() throws IOException, ConnectionException {
+    public Server() throws IOException {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AUndocumentedApi)